src/inference/include/openvino/runtime/intel_myriad/hddl_properties.hpp

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/runtime/properties.hpp>
+
+namespace ov {
+namespace intel_myriad {
+namespace hddl {
+//    RO properties
+/**
+ * @brief Metric to get a int of the device number, String value is METRIC_HDDL_DEVICE_NUM
```

Comment:
metric => property

Reply:
`String value is` comments can be removed everywhere

Reply:
Changed